source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -181,6 +181,7 @@
 CONSTANT = {
     "EMAIL_SERVICE": False,
     "AUTO_ACTIVATE_USER": False,
+    "SEND_EMAIL_CONFIRM_CREATE_PARTICIPATION": False,
```

Comment:
I don't think we should seperate this from "EMAIL_SERVICE". I suggest we do one of these 2 things:
##### First option
Use EMAIL_SERVICE as the only setting related to emails. 
In other words, setting `EMAIL_SERVICE=True` should activate ALL emails in the API. 
##### Second option
Breakdown EMAIL_SERVICE in more specific parameters for every email that can be sent from the API. 
ex: 
```
"EMAIL_SERVICE": {
    "ALL": True, #overrides all other settings
    "ACTIVATION": True,
    "SIGNUP_CONFIRM": True,
    "PARTICIPATION_CONFIRM": True,
}
```
##### Mixed option
We can do option 1 but create an overriding system that can disable some emails. For example, we could use only `"EMAIL_SERVICE"` and when no email template is provided for a scenario (ex: for participations), then no email are sent for that scenario.

---

The real question is: if the administrator uses the email service, will he always use **every** email scenarios? If so, we don't need additional parameters here.

Reply:
I think administrator will not use **every** email scenario but just some subset in function of the organisation's context.

I htink you're right, we need to discuss a real architecture to manage all these settings of email.